discovery/data_sharing/v0.1/data_sharing.md

```diff
@@ -0,0 +1,297 @@
+# "Fediscovery": Data Sharing Specification
+
+## Overview
+
+Specification identifier: `data_sharing`
+
+In order to be able to present results, discovery providers need to be
+able to request content from connected fediverse servers. Two different
+concerns need to be distinguished: Retrieving new content as it is
+created and retrieving historic content to backfill search indexes.
+
+## Restrictions On Content Being Shared
+
+A single FASP will never be used by all fediverse servers. To ensure it
+still gets a broad view not limited to a small number of servers,
+fediverse servers MUST share not only local but also remote content with
+the FASP.
```

Comment:
This may be contentious, especially around the lack of Data Processing Agreements being in place here. i.e., could be a potential legal nightmare.

Reply:
We already mandate that every Specification MUST include this. It is just that I am not sure if this even applies here :confused:

Reply:
I made an issue out of this #36.